### PR TITLE
enable alert task resolve action

### PR DESCRIPTION
**Pull request by ericcf**
[#112389629]

### CODE REVIEW

File: app/models/nurse_task.rb [hunk 1]
```diff
@@ -39,6 +39,14 @@ def overdue?
     Time.zone.now >= overdue_at
   end
 
+  def alert?
+    false
```
Comment (eschlange) [inline app/models/nurse_task.rb]: Is this always supposed to return false?

**Comment by eschlange** [comment]
With regards to the "alert?" method, are the "help_request", "nurse_task", and "lack_of_connectivity_call" using an interface in order to lump these objects together?

**Comment by ericcf** [comment]
`NurseTask` is the superclass (you could call it the interface), so by default any subclasses will just return `false` unless the method is overridden.

**Comment by eschlange** [comment]
Got it. PR looks good, good test coverage.

**Pull request merged**